solr/solr-ref-guide/modules/query-guide/pages/analytics-expression-sources.adoc

```diff
@@ -1,95 +0,0 @@
-= Analytics Expression Sources
```

Comment:
Via `git grep` found "analytics" references in `dev-tools` e.g. https://github.com/apache/solr/blob/df84356a816442c6b2a7b578cb193f22b418cdae/dev-tools/scripts/refguide/new-guide.txt#L75 though not sure if the tooling would automatically handle that i.e. maybe no change needed there.

Reply:
These are data files for the 9.0 transition. The ouput of that script is served in `solr-site` repo as `.htaccess` rules, so this is not affected. Obviously we need to add new rewrite rules when 10.0 is released, for the analytics pages, so they redirect to latest 9.x and not 10.0...

Reply:
I believe that antora should do a redirect to the last known version with that page, but I could be wrong...

Reply:
Ok, cool. We should be able to test that on the nightlies server once this is merged?

Reply:
Kind of, for the nightlies, `latest` still refers to 9.2 (or the current released version), since `10.0.0` is an unreleased version. So `latest` will redirect to the 9.2 page even if the page exists in 10.0.0. We can test it locally by temporarily making `main` a released version and asking it to generate an htaccess file.

I can try that out in a bit

Reply:
Ok this is apparently not a feature, so I've added an issue for antora so we can eventually support it.
https://gitlab.com/antora/antora/-/issues/1060

Hopefully this is implemented by the time Solr 10.0 is released.

Reply:
We can always continue with htaccess as earlier..